phpseclib/Math/BinaryField/Integer.php

```diff
@@ -226,10 +220,9 @@ private static function regularPolynomialMultiply($x, $y)
      *
      * Uses karatsuba multiplication to reduce x-bit multiplications to a series of 32-bit multiplications
      *
-     * @return string
      * @link https://en.wikipedia.org/wiki/Karatsuba_algorithm
      */
-    private static function polynomialMultiply($x, $y)
+    private static function polynomialMultiply($x, $y): string
```

Comment:
Should be able to do `string $x, string $y`

Reply:
Done